pkg/processor/processor.go

```diff
@@ -130,20 +130,24 @@ func (p *CheckProcessor) ProcessMessage(m string) error {
 	}
 
 	p.logger.Trace("Processing vulnerabilities...")
+	p.logger.Debugf("%d vulnerabilities", len(vulns))
```

Comment:
I'm not sure if it makes sense to do these two consecutive logs here, I think it will probably be better to join both with `Debug` level, e.g.: `Processing %d vulnerabilities...`. What do you think?

Reply:
I tried to not touch already existing logs because I added those just for debugging. Then I thought that they were harmless and I kept them. I already mentioned below: do you think it's better to keep them or to remove them?

Reply:
I think we can keep them as they add more information, but I would rearrange them. What do you think?

Reply:
OK, let's do it.

Reply:
Done together